sdk/typescript/src/providers/json-rpc-provider.ts

```diff
@@ -123,8 +123,8 @@ export class JsonRpcProvider extends Provider {
   ) {
     super();
 
-    if (typeof endpoint !== 'string') {
-      this.endpoints = NETWORK_TO_API[endpoint];
+    if ((Object.values(Network) as string[]).includes(endpoint)) {
```

Comment:
@666lcz I noticed this was incorrect from the last PR.

Reply:
Good catch! Thanks for the fix